src/test/java/com/google/sps/UserUtilTest.java

```diff
@@ -19,17 +19,18 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
 import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.ArrayList;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.After;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
-import com.google.sps.UserUtils;
+import com.google.sps.utils.UserUtils;
 
 /** tests the IsValidGameName function */
 @RunWith(JUnit4.class)
-public final class GetEntityFromDatastoreTest {
+public final class UserUtilTest {
   
   private DatastoreService datastore;
```

Comment:
I might have just missed this before, but we shouldn't have global mutable variables in a test like this. If I edit this variable in one test, the state will persist to another test, potentially making for unexpected failures. It's best to just create a new datastore in each test.

Reply:
Okay, that's done for this file. I'll make another pull request to make these changes in the game utils file.